### DISCUSSION TITLE
Unable to parse the log but online log parser says no error

### DISCUSSION THREAD

**Discussion by sunilthemaster** [martin-helmich/prometheus-nginxlog-exporter#243] [Q&A]
Here is my log msg from Nginx
10.96.5.224 - - [27/Jan/2022:08:32:54 +0000] "GET /nginx_status HTTP/1.1" 200 106 "-" "kube-probe/1.20"

And this is the format am passing to exporter
$remote_addr - - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent"

Error msg says
error while parsing line '10.96.5.224 - - [27/Jan/2022:08:32:54 +0000] "GET /nginx_status HTTP/1.1" 200 106 "-" "kube-probe/1.20"': text log parsing err: access log line '10.96.5.224 - - [27/Jan/2022:08:32:54 +0000] "GET /nginx_status HTTP/1.1" 200 106 "-" "kube-probe/1.20"' does not match given format '^'(?P<remote_addr>[^ ]*) - - \[(?P<time_local>[^]]*)\] "(?P<request>[^"]*)" (?P<status>[^ ]*) (?P<body_bytes_sent>[^ ]*) "(?P<http_referer>[^"]*)" "(?P<http_user_agent>[^"]*)"'

Whereas, same test via regex checker is passing
![image](https://user-images.githubusercontent.com/5255193/151322420-a2d5b7af-b237-4dc6-b62f-d8e478323201.png)

Any thoughts/suggestions?

**Comment by martin-helmich** [comment]
Sorry for the late response. This may be just a copy&paste issue, but did I spot an additional `'` in the regex in your error message? I mean the `'` after the first `^` in `'^'(?P<remote_addr>[^ ]) ...`.
